v2/css/styles.css

```diff
@@ -100,7 +100,7 @@ a {
     margin:20px;
 }
 .icon {
-	padding: 10px;
+	padding: 5px;
```

Comment:
Hmmm. This change will have effect on whole screens. 
I'm not in favor of updating it just to fix an issue on Listview screen.

Reply:
@llaske i was thinking adding a specific class to the star icon of listview and then add css changes. it will not effect other screens. example : 
```
.icon {
	padding: 6px 2px 2px 0px;
	width: fit-content;
}

.listview_left .icon {
	padding: 5px !important;
}
```